src/renderer/themes.css

```diff
@@ -95,6 +95,7 @@
   --side-nav-hover-color: #e0e0e0;
   --side-nav-active-color: #757575;
   --search-bar-color: #f5f5f5;
+  --search-list-tag-bg-color: rgb(242 242 242 / 100%);
```

Comment:
**suggestion:** Would `--secondary-card-bg-color` work for this use case? Would prefer not to add new theme variables if possible.

Reply:
![Side-by-side-3](https://github.com/FreeTubeApp/FreeTube/assets/144292954/fdfda4ea-f9cd-4733-bc9f-adf7cf8a1108)

I just had a go at --secondary-card-bg-color. It works fine with every theme apart from black and default. I've attached a screenshot. Middle is with custom background - Left and right are with --secondary-card-bg-color. I'll change it if you think it works for you.

Reply:
That works for me!

Reply:
Ok it's on --secondary-card-bg-color now. The changes to themes.css have been backed out.